### DISCUSSION TITLE
L'Assistente Google ora chiede di sbloccare il dispositivo

### DISCUSSION THREAD

**Discussion by azerbinati** [indomus/forum#3159] [HUB: Home Assistant]
Buongiorno a tutti,
mi chiamo Antonio, e seguo indomus da lungo corso.

Qualche anno fa ho iniziato il mio viaggio nella domotica e posso dire che casa mia la controllo davvero in modo esteso, e grazie a qualche guida di indomus sono riuscito a superare alcuni ostacoli che altrimenti mi avrebbero bloccato (GRAZIE!), tra questi c'è l'integrazione con l'assistente google. 

Non uso nabu, ho configurato home assistant, impostato dns, ssl, google cloud (the hard way), ed ho ottenuto la mia integrazione personale, ormai in funzione da diversi anni.

Allo stato attuale, continua a funzionare tutto, posso chiedere ad esempio di aprire il cancello carrabile, aprire la porta di casa, accendere la pompa di irrigazione, regolare il riscaldamento, le luci, le tapparelle e tante altre cose..

Di recente però google ha attivato Gemini in sostituzione dell'assistente vocale, e c'è un piccolo cambiamento molto fastidioso: viene ora richiesto di sbloccare il dispositivo.

La scena è questa, arrivo a casa in auto, chiedo a google di aprirmi il cancello.. e in risposta mi ritrovo, "perfetto, ma devi prima sbloccare il dispositivo". 

Ora, se devo prendere in mano il telefono e sbloccarlo, viene meno tutta la comodità del comando vocale. (Ah, esempio hackerato, se qualcuno lo ha fatto, sa che google chiede il pin per aprire un cancello, in realtà io gli chiedo di chiuderlo, così non chiede il pin, ma l'effetto è che cmq viene dato l'impulso apri/ferma/chiudi sul pin della centralina tramite uno shelly 1, ed il cancello si apre anche se chiedo di chiuderlo, scusato piccolo OT ma altrimenti qualcuno potrebbe pensare che avevo rimosso la richiesta del PIN e fare confusione sul tipo di "sblocco" richiesto.)

Per risolvere, potrei togliere Gemini (cosa non impossibile visto che le risposte sono qualitativamente scarse, ma magari tra 3 mesi ritorno a questo punto) o configurare che il telefono non si blocchi quando sono a casa (ma i bambini chissà che cosa ci farebbero se lo lascio in giro).. inoltre alcuni comandi li voglio lanciare anche se mi trovo lontano da casa.

Qualcuno si è imbattuto ed è magari riuscito a risolvere questo problema?
Grazie

**Comment by Eddy-tao** [comment]
Ciao, al momento l'unico modo per sbloccare la situazione è quello di attivare il riconoscimento vocale. 
Quando dici Ok Google, il sistema riconosce la tua voce e sblocca la rubrica oppure le varie skill.
Riconoscendo la voce non ti chiede più di sbloccare il dispositivo perché lo hai già sbloccato con la voce però solo se dici il comando Ok Google
